wifiphisher/common/interfaces.py

```diff
@@ -517,6 +551,9 @@ def get_interface(self, has_ap_mode=None, has_monitor_mode=None):
 
         # return interface if found otherwise raise an error
         if chosen_interface:
+            adapter = self._name_to_object[chosen_interface]
+            if adapter.is_managed_by_nm:
```

Comment:
If we go with what you purpose we would not be able to distinguish between when we can't find an interface and when the interface found was managed by network manager.